rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -159,6 +155,29 @@ public static boolean scriptCorrectlySpendsTx(BtcTransaction tx, int index, Scri
         }
     }
 
+    /**
+     * @param amountSent      amount sent
+     * @param activations     the network HF activations configuration
+     * @param bridgeConstants the Bridge constants
+     * @return true if amount sent is over the minimum for pegin transaction
+     */
+    public static boolean isAmountSentOverMinimum(Coin amountSent, ActivationConfig.ForBlock activations, BridgeConstants bridgeConstants) {
+        Coin minimumPegInTxValue = getMinimumPegInTxValue(activations, bridgeConstants);
+        return !amountSent.isLessThan(minimumPegInTxValue);
+    }
+
+    /**
+     * @param activations     the network HF activations configuration
+     * @param bridgeConstants the Bridge constants
+     * @return the minimum amount value defined for pegin transaction
+     */
+    public static Coin getMinimumPegInTxValue(ActivationConfig.ForBlock activations, BridgeConstants bridgeConstants) {
```

Comment:
Could you make use of this new method in co.rsk.peg.BridgeUtils#isValidPegInTx(co.rsk.bitcoinj.core.BtcTransaction, java.util.List<co.rsk.peg.Federation>, co.rsk.bitcoinj.script.Script, co.rsk.bitcoinj.core.Context, co.rsk.config.BridgeConstants, org.ethereum.config.blockchain.upgrades.ActivationConfig.ForBlock) please?